app/schemas.py

```diff
@@ -448,6 +448,22 @@ class Meta(BaseSchema.Meta):
         exclude = tuple(set(BaseTemplateSchema.Meta.exclude) - {"jobs"})
 
 
+class TemplateEmailFilesSchema(BaseSchema):
+    filename = field_for(models.TemplateEmailFile, "filename", required=True)
+    link_text = field_for(models.TemplateEmailFile, "link_text", required=False)
+    retention_period = field_for(models.TemplateEmailFile, "retention_period", required=False)
+    validate_users_email = field_for(models.TemplateEmailFile, "validate_users_email", required=True)
```

Comment:
I wonder - why is `validate_users_email` required, but `retention_period` is not? Aren't they both attributes that would always be set in the database, but they would have a default value if not provided?

Reply:
changed